rules/windows/other/win_wmi_persistence.yml

```diff
@@ -1,33 +1,51 @@
+action: global
 title: WMI Persistence
 id: 0b7889b4-5577-4521-a60a-3376ee7f9f7b
 status: experimental
-description: Detects suspicious WMI event filter and command line event consumer based on event id 5861 and 5859 (Windows 10, 2012 and higher)
-author: Florian Roth
+description: Detects suspicious WMI event filter and command line event consumer based on WMI and Security Logs.
+author: Florian Roth, Gleb Sukhodolskiy, Timur Zinniatullin oscd.community
 date: 2017/08/22
-modified: 2020/08/23
+modified: 2020/10/13
 references:
     - https://twitter.com/mattifestation/status/899646620148539397
     - https://www.eideon.com/2018-03-02-THL03-WMIBackdoors/
+    - https://attack.mitre.org/techniques/T1546/003/
```

Comment:
Hello @GlebSukhodolskiy! could you please remove the ATT&CK tag from references, as the technique has already been mentioned in the `tags`?

Reply:
Sure, done :)

Reply:
Thanks!